renovate.json

```diff
@@ -1,5 +1,45 @@
 {
-  "extends": [
-    "config:base"
+  "pinVersions": false,
+  "semanticCommits": true,
+  "semanticPrefix": "build:",
+  "commitMessage": "{{semanticPrefix}}update {{depName}} to version {{newVersion}}",
+  "separateMajorReleases": false,
+  "dependencies": {
+    "semanticPrefix": "fix:"
+  },
+  "packageFiles": [
+    "package.json"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["^@angular\/.*"],
+      "groupName": "angular",
+      "pinVersions": false
+    },
+    {
+      "packagePatterns": ["^@commitlint\/.*"],
+      "groupName": "commitlint",
+      "pinVersions": false
+    },
+    {
+      "packagePatterns": ["^@types\/.*"],
+      "groupName": "type definitions",
+      "pinVersions": false
+    },
+    {
+      "packagePatterns": ["^jasmine.*"],
+      "groupName": "jasmine",
+      "pinVersions": false
+    },
+    {
+      "packagePatterns": ["^karma.*"],
+      "groupName": "karma",
+      "pinVersions": false
+    },
+    {
+      "packagePatterns": ["^protractor.*"],
+      "groupName": "protractor",
+      "pinVersions": false
+    }
   ]
 }
```

Comment:
@dherges I think you need your own preset for this :)

Reply:
Yes, I was thinking for an angular config preset. Is https://github.com/singapore/renovate-config/tree/master/packages still a good place for such?

Reply:
Well, I was more meaning your own personal preset, e.g. `renovate-config-dherges` if you use the same across all repos or `renovate-config-spektrakel-blog` even if only for these repos.

However other presets are welcomed to that repo if you think they're of benefit to others. Beware in some cases the "package.json" is autogenerated from a script so you may need to add to the script in order to generate the json.. or just paste code or pseudocode into an issue and I can do it.

There are already presets for angular 1.x and angular 2+ monorepos. If you add the preset "group:monorepos" to your config then you'll get the benefit of all the monorepos I've already caught and defined.